src/dmd/backend/dwarf.c

```diff
@@ -234,7 +234,7 @@ void dwarf_appreladdr(int seg, Outbuffer *buf, int targseg, targ_size_t val)
     if (I64)
     {
         dwarf_addrel64(seg, buf->size(), targseg, val);
-        buf->write64(0);
+        write64(buf, 0);
```

Comment:
Why this change?

Reply:
It should work correctly if you use `stdint.h` and `int64_t` on the C side. I've encountered this problem before with divcoeff.d and cod2.c.